### ISSUE TITLE
The specific implementation of MeME

### ISSUE THREAD

**Issue by Ni-Songhao** [supertyd/XTrack#2]
Hello, I'd like to know how the gate noise is added in the MeME module, and how the load loss is calculated. Is the calculation method of its load loss similar to that of sMOE( Outrageously Large Neural Networks: The Sparsely-Gated Mixture-of-Experts Layer)? And the demonstration and the paper have shown the X modality to improve RGB. However, the RGB to improve X modlaity has not been explained. Also, do RGB and the X modality share the same MeME? Does the classification of its experts not take RGB into account?. Is it possible to publicly release part of the module code first, and then update the complete and training code later?

**Comment by Ni-Songhao** [comment]
> Hi, my implementation of the Mixture of Experts (MoE) is based on the following repository: [mixture-of-experts](https://github.com/davidmrau/mixture-of-experts). I hope you find it useful. In my approach, both RGB and X modalities share the same MeME . Unlike Onetracker, we did not utilize LoRA for fine-tuning the RGB Tracker.
> 
> The RGB modality is always available, while X can only utilize one of the T/D/E modalities at a time. To address this constraint, we designed this MoE.

Thank you for your reply. In the paper, you mentioned that RGB also improves X modalities, but the examples in the paper all use X to improve RGB. So, RGB tokens will also be assigned to T/D/E.? In this case, how is the route classification loss calculated? The labels do not contain the RGB.
